go.mod

```diff
@@ -12,9 +12,9 @@ require (
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.2.0
 	github.com/containers/buildah v1.29.1-0.20230201192322-e56eb25575c7
-	github.com/containers/common v0.51.1-0.20230221111605-852ca05a1fbb
+	github.com/containers/common v0.51.1-0.20230222133614-9a4f00f2abb2
 	github.com/containers/conmon v2.0.20+incompatible
-	github.com/containers/image/v5 v5.24.2-0.20230221092641-10858b2058d8
+	github.com/containers/image/v5 v5.24.2
```

Comment:
This is a downgrade.

Reply:
sigh ... stupid Go.

Updating c/image in c/common does the following:
```
common (main) $ go get github.com/containers/image/v5@10858b2058d8
panic: internal error: can't find reason for requirement on github.com/google/pprof@v0.0.0-20210407192527-94a9f03dee38

goroutine 1 [running]:
cmd/go/internal/modget.(*resolver).updateBuildList.func1({{0xc000703230?, 0xc000179620?}, {0xc000030c30?, 0xc0004a9630?}})
        /usr/lib/golang/src/cmd/go/internal/modget/get.go:1760 +0x114
cmd/go/internal/modget.(*resolver).updateBuildList(0xc000708900, {0xb313b0, 0xc0000280b8}, {0x0, 0x0, 0x0})
        /usr/lib/golang/src/cmd/go/internal/modget/get.go:1765 +0x597
cmd/go/internal/modget.(*resolver).resolveQueries(0xc000708900, {0xb313b0, 0xc0000280b8}, {0xc000014050, 0x1, 0x472485?})
        /usr/lib/golang/src/cmd/go/internal/modget/get.go:1243 +0x1e5
cmd/go/internal/modget.runGet({0xb313b0, 0xc0000280b8}, 0xc00002c510?, {0xc0000241a0, 0x1, 0x1})
        /usr/lib/golang/src/cmd/go/internal/modget/get.go:314 +0x40b
main.invoke(0xe2d000, {0xc000024190, 0x2, 0x2})
        /usr/lib/golang/src/cmd/go/main.go:225 +0x34e
main.main()
        /usr/lib/golang/src/cmd/go/main.go:179 +0x7d1
```

Reply:
what?! how is this even possible